airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/source.py

```diff
@@ -431,6 +455,9 @@ def path(
     def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
         yield response.json()
 
+    def get_error_mapping(self):
+        return get_google_analytics_data_api_metadata_error_mapping(self.config.get("property_id"))
+
```

Comment:
e.- 
_get_error_mapping_ is implemented in **GoogleAnalyticsDataApiMetadataStream**. 

Previously, _check_connection_ in **SourceGoogleAnalyticsDataApi** would format and raise an "Incorrect Property ID:..." message for 400, 403 HTTP exceptions checking the connection (read records). Now the method _get_google_analytics_data_api_metadata_error_mapping_ handles this task.